src/components/pages/Analytics/Graph.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import axios from 'axios';
+import embed from 'vega-embed';
+
+export default function Graph() {
+  let url =
```

Comment:
If you want to make this component more general use then I might make `url` a prop that gets passed in